apps/bfd-model/bfd-model-rif-samples/src/test/java/gov/cms/bfd/model/rif/samples/SampleDataColumnsTest.java

```diff
@@ -127,9 +129,12 @@ private static Enum<?>[] getColumnsInEnum(RifFileType rifFileType) {
   }
 
   /**
-   * @param columns the array of columns to format
+   * Appeneds the columns together using a pipe delimiter and returns it as a String.
+   *
+   * @param columns the array of columns to format {@link T}
    * @param columnNameFunction the {@link Function} to convert each column entry to the column name
-   * @return the specified columns, but in the format used in RIF header rows
+   * @return the specified columns {@link T}, but in the format used in RIF header rows
+   * @param <T> to header format
```

Comment:
Only curious... is this formatting/ordering correct when specifying the generic?

Reply:
I believe so

Reply:
I had a similar question earlier on if {@link T} actually works. The param formatting is fine, but actually yeah the return should be the last item.

Reply:
This was _resolved_ but appears to remain unaddressed. Can you take another pass?

Reply:
done